src/context/index.tsx

```diff
@@ -55,12 +94,15 @@ export const FoundryProvider = ({
     ...colorsEnum,
     ...colors,
   };
+  const mergedAnalytics =
+    defaultAnalyticsFunction === analyticsFunction ? defaultAnalyticsFunction : analyticsFunction;
```

Comment:
is this backwards?

Reply:
This reads the same as `const mergedAnalytics = analyticsFunction`, no?

Reply:
I removed it entirely and passed analyticsFunction directly. Not sure what I was doing here.